src/models/meta_details.rs

```diff
@@ -355,6 +383,74 @@ fn streams_update<E: Env + 'static>(
     }
 }
 
+fn suggested_stream_update(
```

Comment:
Could you at least document the function so that we know what it's suppose to do in case of a bug, we could track that logic (intended vs actual)

Reply:
What format would you suggest for this. as other methods are not documented?

Reply:
A lot of the information for this method was provided in the PR comment. Use that and include that at this point we're talking about the `Next video id` while binge watching.

Reply:
Small tweak I've made is to use the actual Doc comments `///` that will show on the documentation instead of regular comments `//`